Apps/Sandcastle/gallery/Particle System Tails.html

```diff
@@ -0,0 +1,228 @@
+        particleCanvas.height = 20;
+        var context2D = particleCanvas.getContext('2d');
+        context2D.beginPath();
+        context2D.arc(8, 8, 8, 0, Cesium.Math.TWO_PI, true);
+        context2D.closePath();
+        context2D.fillStyle = 'rgb(255, 255, 255)';
+        context2D.fill();
+    }
+    return particleCanvas;
+}
+
+// Add plane to scene
+var hpr = new Cesium.HeadingPitchRoll(0.0, Cesium.Math.PI_OVER_TWO, 0.0);
+var orientation = Cesium.Transforms.headingPitchRollQuaternion(planePosition, hpr);
+var entity = viewer.entities.add({
+    model : {
+        uri : '../../SampleData/models/CesiumAir/Cesium_Air.gltf',
+        scale : 3.5
+    },
+    position : planePosition,
+    orientation : orientation
+});
+
+// creating particles model matrix
+var transl = Cesium.Matrix4.fromTranslation(particlesOffset, new Cesium.Matrix4());
+var translPosition = Cesium.Matrix4.fromTranslation(planePosition, new Cesium.Matrix4());
+var particlesModelMatrix = Cesium.Matrix4.multiplyTransformation(translPosition, transl, new Cesium.Matrix4());
+
+// creating the particle systems
+var rocketOptions = {
+    numberOfSystems : 50.0,
+    iterationOffset :  0.1,
+    cartographicStep : 0.000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        minimumRed : 1.0,
+        green : 0.5,
+        minimumBlue : 0.05,
+        alpha : 1.0
+    }, {
+        red : 0.9,
+        minimumGreen : 0.6,
+        minimumBlue : 0.01,
+        alpha : 1.0
+    }, {
+        red : 0.8,
+        green : 0.05,
+        minimumBlue : 0.09,
+        alpha : 1.0
+    }, {
+        minimumRed : 1,
+        minimumGreen : 0.05,
+        blue : 0.09,
+        alpha : 1.0
+    }]
+};
+
+var cometOptions = {
+    numberOfSystems : 100.0,
+    iterationOffset : 0.003,
+    cartographicStep : 0.0000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        red : 0.6,
+        green : 0.6,
+        blue : 0.6,
+        alpha : 1.0
+    }, {
+        red : 0.6,
+        green : 0.6,
+        blue : 0.9,
+        alpha : 0.9
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.7,
+        alpha : 0.5
+    }]
+};
+
+var scratchCartesian3 = new Cesium.Cartesian3();
+var scratchCartographic = new Cesium.Cartographic();
+var forceFunction = function(options, iteration) {
+    var iterationOffset = iteration;
+    var func = function(particle) {
+        scratchCartesian3 = Cesium.Cartesian3.normalize(particle.position, new Cesium.Cartesian3());
+        scratchCartesian3 = Cesium.Cartesian3.multiplyByScalar(scratchCartesian3, -1.0, scratchCartesian3);
+
+        particle.position = Cesium.Cartesian3.add(particle.position, scratchCartesian3, particle.position);
+
+        scratchCartographic = Cesium.Cartographic.fromCartesian(particle.position, Cesium.Ellipsoid.WGS84, scratchCartographic);
+
+        var angle = Cesium.Math.PI * 2.0 * iterationOffset / options.numberOfSystems;
+        iterationOffset += options.iterationOffset;
```

Comment:
why are we saving

`var iterationOffset = iteration;`

then increment this new variable, and then not doing anything with it? Unless I'm missing something, I think we can remove that and just use `iteration` direction in the above angle calculation.